code/modules/mob/say.dm

```diff
@@ -145,6 +145,29 @@
 
 	return null
 
+/**
+  * Replaces all brackets from the input with italics
+  *
+  * Requires an equal number of opening and closing brackets, to avoid overflow.
+  */
+/mob/proc/say_emphasis(input)
+	/// Number of opening brackets in the input
```

Comment:
Triple / applies to class based var definitions, not in-proc ones. Please remove.